client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -532,6 +532,7 @@ export const FeatureRow = React.memo(
     ) : (
       <code>{name}</code>
     );
+    const borderleftwidth = `calc(8 * ${(name.match(/\./g) || []).length}px)`;
```

Comment:
Let's give that calculated value a name, use `String.split` for simplicity, and ...

```suggestion
    const path = name.split(".")
    const depth = path.length;
```

(Depth in favor of level, because level already has a meaning in web standards.)